certification/engine/operatorsdk.go

```diff
@@ -0,0 +1,188 @@
+package engine
```

Comment:
This is probably a broader discussion on the code design: we have two engine directories atm; I believe we should be more prescriptive about which one should be used for the preflight internal. My intuition is to use the certification/internal/engine for everything which is internal to preflight (such as operatorsdk, openshift, crane) and let the certification/engine be for the check engine. I would like to hear everyone's feedback on this matter. 

@komish @bcrochet @acornett21 @jomkz

Reply:
I will test whether this can continue to live in the internal directory. If that's the case, then it's probably a moot point, and I'll just leave it there.

Reply:
And... it can. I'll move this to internal.